paths/locales/download.yaml

```diff
@@ -73,6 +73,8 @@ parameters:
     schema:
       type: object
       properties: {}
+    style: deepObject
+    explode: true
```

Comment:
This can be removed? The current solution does not rely on these properties, tested locally.